src/config.cpp

```diff
@@ -59,6 +66,109 @@ void parse_uint32(void* value_ptr, const char* txt) {
     cfgval->set = true;
 }
 
+// Helper function to trim whitespace and remove quotations
+std::string trim(const std::string& str) {
+    size_t first = str.find_first_not_of(" \t\n"); // Also include quotes
```

Comment:
The comment does not match the content

Reply:
The comment mentions "quotes" while they are not part of the argument of `find_first_not_of`.